changelog/issue-niSSIctzX93Fh5QmnvSE2T8QnUhwF19a.md

```diff
@@ -0,0 +1,3 @@
+level: silent
+reference: issue niSSIctzX93Fh5QmnvSE2T8QnUhwF19a
```

Comment:
that's a weird-looking issue id!  Did `--no-bug` produce this?

Reply:
I wasn't aware of this. I used a random string bash function. I'll use --no-bug in the future.